feat: add new devfact article (#303)
BedrockStreaming/tech.bedrockstreaming.com: _posts/2022-03-26-bedrock-dev-facts-16.md

> 
> E: La meilleure extension de neovim est le bloc-note.
> 
> F: Mon bloc-note c'est VSCode.

# Quand on va à la pêche à l’exception

```
Exception message (1) :
------------------------
Attempted to load class "TruiteException" from namespace "Bedrock\Stores\Infra\HttpClient".
Did you forget a "use" statement for another namespace?
```

# _"Ce n'est pas un échec... ça n'a pas marché"_

> Y'a pas de bug, y'a juste un truc qui fonctionne pas

# Les bottleneck c’est la vie

> La vie est un bottleneck

# Atlas, le géant ? 

> Qui c'est qui a déplacé la terre ?

# Traquilou bilou

> Le pair programming n’est pas du programming pépère

# Tester c'est ...

> A: Faire des tests, c'est quand même moins chiant à faire à deux
>
> B: Ouais, comme le sexe

# Labyrinthe

> Pourquoi vous vous faites chier à mettre les secrets dans SOPS. 
> Vous l'auriez mis dans votre doc, personne ne les aurait trouvé.

# Une trouvaille de twitter

{% twitter https://twitter.com/maxime/status/1483392747667136514 %}

# Voilà qui voilà !

> A: Je crois que la branche master elle est pas protégée.
> 
> ...
> 
> Ah bah heureusement que je suis la pour tester
> 

> > B: Le plus étonnant c'est que 'A' est le droit de push non ?

Reviewer: ```suggestion
> B: Le plus étonnant c'est que 'A' ait le droit de push non ?
```

> 
> A: Le plus étonnant c'est que j'ai quelque chose à push non ? 
> 
> ...
>
> On a résolu une faille de sécurité sur le projet, grâce à mon investigation.
> 
> C: Le mec c'est inspecteur gadget enfaîte, on sait pas comment mais il finit par être utile. 

# 🕯

> The Domain is a sanctuary

# Quand il y a trop de viennoiserie chez Bedrock

> A: Fait gaffes tu es en ordre croissant, l'ordre pain au chocolat est bien meilleur

# Mais oui c'est clair !

> On complexifie pour faire plus simple !

# La PRO-crastination

> Lead: C'était qui le chargé du monitoring hier ?

> > A: C'était moi mais j'ai rien fait, mais si vous voulez je peux le refaire aujourd'hui.
> > B: Si tu veux je peux même t'aider à rien refaire.

Reviewer: ```suggestion

> A: C'était moi mais j'ai rien fait, mais si vous voulez je peux le refaire aujourd'hui.

> B: Si tu veux je peux même t'aider à rien refaire.
```

parce que les sauts de lignes n'apparaissent pas sinon

# Ça sent un peu la poussière ici

> A: Eh eh, là on a effectivement sur-gonflé l'estim ... (mais le côté pas dans l'inconnu l'explique)

> > B: Ouais c'était carrément la peur de toucher à un projet legacy de chez legacy, qu'on ne connait pas, pas dans notre scope...
> > A: Mais pas de souci majeur pour un développeur qui pratiquait Symfony sous Pompidou (le code date de cette époque)

Reviewer: ```suggestion

> B: Ouais c'était carrément la peur de toucher à un projet legacy de chez legacy, qu'on ne connait pas, pas dans notre scope...

> A: Mais pas de souci majeur pour un développeur qui pratiquait Symfony sous Pompidou (le code date de cette époque)
```

same

---
layout: post
title: "Bedrock Dev Facts #16"
description: "Les plus belles phrases de nos équipes techniques"
author: bedrock 
category: 
tags: [devfacts, humour]
color: rgb(251,87,66)
---

Sur ce début d'année les équipes de Bedrock se sont lachées ça promet pour le reste de l'année.
[Les autres articles de cette série sont disponibles ici.](/tags/#devfacts)

# Promis !

> ![Screenshot montrant un échange de message slack indiquant qu'une personne notifie qu'un de ses collègue à indiqué dans un message épinglé agé d'un ans que cela n'était pas encore fait. Et l'individu rajouta "Demain promis"](/images/posts/dev-facts-16/slack-screenshot.png)

Reviewer 2: Petites typos /à/a/ /collègue/collègues/, j’en ai profité pour retravailler la phrase.

```suggestion
![Screenshot montrant un échange de message slack : une personne notifie qu’un de ses collègues a indiqué dans un message épinglé il y a un an que cela n’était pas encore fait. Et l'individu rajouta « Demain promis »](/images/posts/dev-facts-16/slack-screenshot.png)
```

# L'arbre qui cache la forêt

> Quand c'est la couleur des noms de fichiers qui te permet de te repérer dans l'arborescence du projet… 

> ![Screenshot d'une arborescence de fichiers ou trois fichiers ont le même nom dans trois doissiers imbriqués](/images/posts/dev-facts-16/arbo-fichiers.png)

Reviewer 2: ```suggestion
![Screenshot d'une arborescence de fichiers où trois fichiers ont le même nom dans trois doissiers imbriqués](/images/posts/dev-facts-16/arbo-fichiers.png)
```

# La boucle a bouclé

Le débat est et sera éternel.

> A: La meilleure extension de VSCode c'est d'installer Webstorm.
> 
> B: Et la meilleure extension de Webstorm c'est d'installer Phpstorm.
> 
> C: Et la meilleure extension de Phpstorm c'est d'installer vim.
> 
> D: La meilleure extension de Vim c’est d’installer Neovim.
> 
> E: La meilleure extension de neovim est le bloc-note.
> 
> F: Mon bloc-note c'est VSCode.

# Quand on va à la pêche à l’exception

```
Exception message (1) :
------------------------
Attempted to load class "TruiteException" from namespace "Bedrock\Stores\Infra\HttpClient".
Did you forget a "use" statement for another namespace?
```

# _"Ce n'est pas un échec... ça n'a pas marché"_

> Y'a pas de bug, y'a juste un truc qui fonctionne pas

# Les bottleneck c’est la vie

> La vie est un bottleneck

# Atlas, le géant ? 

> Qui c'est qui a déplacé la terre ?

> # Traquilou bilou

Reviewer 3: ```suggestion
# Tranquilou bilou
```

> Le pair programming n’est pas du programming pépère

# Tester c'est ...

> A: Faire des tests, c'est quand même moins chiant à faire à deux
>
> B: Ouais, comme le sexe

# Labyrinthe

> Pourquoi vous vous faites chier à mettre les secrets dans SOPS. 
> Vous l'auriez mis dans votre doc, personne ne les aurait trouvé.

# Une trouvaille de twitter

{% twitter https://twitter.com/maxime/status/1483392747667136514 %}

# Voilà qui voilà !

> A: Je crois que la branche master elle est pas protégée.
> 
> ...
> 

> > Ah bah heureusement que je suis la pour tester

Reviewer 4: ```suggestion
> Ah bah heureusement que je suis là pour tester
```

> 
> B: Le plus étonnant c'est que 'A' ait le droit de push non ?
> 
> A: Le plus étonnant c'est que j'ai quelque chose à push non ? 
> 
> ...
>
> On a résolu une faille de sécurité sur le projet, grâce à mon investigation.
> 

> > C: Le mec c'est inspecteur gadget enfaîte, on sait pas comment mais il finit par être utile. 

Reviewer 2: ```suggestion
> C: Le mec c'est inspecteur gadget en fait, on sait pas comment mais il finit par être utile. 
```
